packages/turf-line-intersect/package.json

```diff
@@ -63,11 +63,8 @@
     "write-json-file": "*"
   },
   "dependencies": {
-    "@turf/geojson-rbush": "^3.2.0",
     "@turf/helpers": "^6.5.0",
-    "@turf/invariant": "^6.5.0",
-    "@turf/line-segment": "^6.5.0",
-    "@turf/meta": "^6.5.0",
+    "sweepline-intersections": "^1.3.1",
```

Comment:
Gotta bump this to 1.4.0 and yarn install again. Hopefully I got the change right :)

Reply:
See my comment for further improving the type signature - https://github.com/rowanwins/sweepline-intersections/pull/12/files#r831527331